casanovo/denovo/dataloaders.py

```
@@ -342,6 +355,288 @@ def _make_dataset(
+                                            ] = max(lower_offset, upper_offset)
+                                        prec_to_spec[(mz, rt, charge)][
+                                            "scans"
+                                        ].append(
+                                            [x for x in zip(mzs, intensities)]
+                                        )
+                                        prec_to_spec[(mz, rt, charge)][
+                                            "rts"
+                                        ].append(cur_rt - rt)
+        return prec_to_spec
+
+    def _get_centers(mzml_file):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_get_centers`

(ANN202)